lib/hyrax/configuration.rb

```diff
@@ -402,6 +402,14 @@ def iiif_image_size_default
     end
     attr_writer :iiif_image_size_default
 
+    # IIIF metadata - fields to display in the metadata section
+    #
+    # @return [#Array] fields
+    def iiif_metadata_default
```

Comment:
Would you mind renaming this to iiif_metadata_fields?

Reply:
Done